### ISSUE TITLE
Inconsistent results: docker container vs local machine

### ISSUE THREAD

**Issue by olalonde** [PyCQA/pyflakes#284]
In Docker:

```bash
$ flake8 --version
3.3.0 (mccabe: 0.6.1, pycodestyle: 2.3.1, pyflakes: 1.5.0) CPython 3.5.2 on Linux
$ flake8
./rise/engines/base.py:54:17: E999 SyntaxError: invalid syntax
./rise/engines/bow/bow.py:62:17: E999 SyntaxError: invalid syntax
$ echo $?
1
```

On my local machine:

```bash
$ flake8 --version
3.3.0 (mccabe: 0.6.1, pycodestyle: 2.3.1, pyflakes: 1.5.0) CPython  3.6.2 on Darwin
$ flake8
$ echo $?
0
```

I was wondering what could be causing the disparity.

**Comment by olalonde** [comment]
Well I found out... both syntax errors had this form:

```
def somefn(
  something,
  **kwargs,
):
  pass
```

I had to remove the trailing `,` after `**kwargs`. Apparently this is allowed in python 3.6.2 but not in python 3.5.2? Anyways, problem solved!

**Comment by asmeurer** [comment]
I've [hit that before too](https://stackoverflow.com/questions/16950394/python-why-is-this-invalid-syntax). Apparently it was a quirk in the grammar that was fixed in 3.6.
